### PR TITLE
Replace and rename nightly email files to fix errors in horizon

### PR DESCRIPTION
**Pull request by bhavanijayakumaran**

### CODE REVIEW

File: util/cron/nightly_email.pl [hunk 1]
```diff
@@ -68,9 +70,11 @@
 
     $summary = "Tests run: $cursucc Successes ($delsucc), $curfail Failures ($delfail)";
 } else {
-    $summary = "Tests run: failed";
+    $summary = "Tests run: failed new failures";
```
Comment (jhh67) [inline util/cron/nightly_email.pl:73]: This should probaby say "Tests run: new failures".

File: util/cron/nightly_email.pl [hunk 2]
```diff
@@ -167,3 +141,8 @@
         system("cp -pv $sortedsummary $prevsummary");
     }
 }
+
+return(0);
```
Comment (jhh67) [inline util/cron/nightly_email.pl:145]: The indentation is incorrect.

**Review by jhh67 — CHANGES_REQUESTED** [review]

File: util/cron/nightly_email.pl [hunk 3]
```diff
@@ -167,3 +141,8 @@
...
+
+}
```
Comment (tzinsky) [inline util/cron/nightly_email.pl:147]: Does this resolve this error:   Unmatched right curly bracket at /ptmp/jenkins/chapel-ci/workspace/test-email/util/cron/nightly_email.pl line 147, at end of line
syntax error at /ptmp/jenkins/chapel-ci/workspace/test-email/util/cron/nightly_email.pl line 147, near "}" in this run:  https://chapel-ci.us.cray.com/job/test-email/82/console

Comment (bhavanijayakumaran) [reply util/cron/nightly_email.pl]: Removed in the latest patch

File: util/cron/nightly_email.pl (hunk 2 above)
Comment (bhavanijayakumaran) [reply util/cron/nightly_email.pl]: removed the return statement

File: util/cron/nightly [hunk 4]
```diff
@@ -878,9 +878,9 @@ if ($runtests == 0) {
     }
 
 # FIXME: Pass correct args here!
-     # `$chplhomedir/util/cron/nightly_email.pl $status "$rawsummary" "$sortedsummary" "$prevsummary" "$mailer" "$nochangerecipient" "$recipient" "$subjectid" "$config_name" "$revision" "$rawlog" "$starttime" "$endtime" "$crontab" "$testdirs" $debug`;
+   `$chplhomedir/util/cron/nightly_email.pl $status "$rawsummary" "$sortedsummary" "$prevsummary" "$mailer" "$nochangerecipient" "$recipient" "$subjectid" "$config_name" "$revision" "$rawlog" "$starttime" "$endtime" "$crontab" "$testdirs" $debug`;
```
Comment (jhh67) [inline util/cron/nightly:881]: The backtick '\`' character in Perl causes it to run the external command and return the output. In this case, we aren't doing anything with the output, so it just disappears. If you change this to ``print `$chplhome... ``then any output from `nightly_email.pl` will be printed by the `nightly` script, which will help us debug it.

**Comment by jhh67** [comment]
I figured out how to run `nightly` as a stand-alone program, meaning you can run and debug it on your laptop. Here is the process.
```
> clone the Chapel repository
> export CHPL_NIGHTLY_DEBUG_EMAIL=john.hartman@hpe.com # must be set, but no email will be sent
> export CHPL_NIGHTLY_CRON_LOGDIR=/tmp
> util/cron/nightly -debug -hello
```
The last line runs only `test/release/examples/hello.chpl`. You can change `hello.good` if you want the test to fail, or `hello.chpl` if you want the compile to fail. In both cases `email.txt` should be created. I'm not sure how to trigger a success after either a test or compile failure (which should also create `email.txt`). I'll look into that.

**Pull request closed without merging**